tests/workbenches/conftest.py

```diff
@@ -62,6 +76,17 @@ def default_notebook(
     # Set the correct username
     username = get_username(dyn_client=admin_client)
 
+    # Set the image path based on internal image registry status
+    try:
+        internal_image_registry.wait_for_condition(
+            condition=ClusterOperator.Condition.AVAILABLE,
+            status=ClusterOperator.Condition.Status.TRUE,
+            timeout=Timeout.TIMEOUT_30SEC,
+        )
+        minimal_image_path = f"{INTERNAL_IMAGE_REGISTRY_PATH}/{py_config['applications_namespace']}/{minimal_image}"
+    except (ResourceNotFoundError, TimeoutExpiredError):
+        minimal_image_path = ":" + minimal_image.rsplit(":", maxsplit=1)[1]
```

Comment:
Do you want to log a warning in this case?

Reply:
Not really, it's an expected way the test may go. On disconnected cluster.